budget/models.py

```diff
@@ -61,9 +61,9 @@ def get_transactions_to_settle_bill(self):
         credits, debts, transactions = [],[],[]
         # Create lists of credits and debts
         for person in self.members:
-            if balance[person.id] > 0:
+            if round(balance[person.id], 2) > 0:
```

Comment:
This is what I was also thinking, but I decided against it since `balance()` seems to be used in a lot of other places.  Some calling functions may need the full floating-point precision to work.

It's a bit like unicode vs. bytestrings: always use unicode internally, and only decode/encode to UTF-8 bytestrings when handling inputs/ouputs.